### ISSUE TITLE
Переформатирование информации об автомобиле при поиске по vin

### ISSUE THREAD

**Issue by makovsv** [makovpv/Carzine#46]
Проблема, что мы очень нерационально пространство при поиске по ВИН.
Убираем картинку с авто - так как она часто может не соответствовать реальному внешнему виду и плохо влиять на конвертацию пользователя
В начале пишем марку и модель. Например Nissan X-trail T-31 и дальше компактно меньшим шрифтом: BODY: WAGON(W); ENGINE: MR20DE TYPE ENGINE(MR20DE); AREA: EUR/C(EUR/C); WHL DRV: 4 WHEEL DRIVE(4WD); GRADE: LE GRADE(LE); TRANS: CVT(CVT)

![Image](https://github.com/makovpv/Carzine/assets/38849646/7ed54bdb-ad21-4aa2-b395-dc154245025c)

**Comment by makovpv** [comment]
сейчас на сайте отображается три параметра
- марка
- модель
- модификация

например для фокуса это соответственно
марка = Ford
модель = Focus / Focus C-Max
модификация = Focus CB4 2008-2011 FWD

если я правильно понял ты хочешь убрать отображение информации о модели (оставить только марка + модификация)
т.е. для фокуса будет строка "Ford Focus CB4 2008-2011 FWD"

**Comment by makovsv** [comment]
Нет, тогда оставляем все. Пишем через запятую: марка, модель, модификация
